addons/udes_stock/models/stock_picking.py

```diff
@@ -296,57 +332,74 @@ def _backorder_move_lines(self):
         self.ensure_one()
 
         moves = self.move_lines
```

Comment:
Super nit:
It might seems ridiculous but as much as this variable is supposed to make the code shorter and simpler I keep coming back to read again what "moves" are as the name does not suggest much and I have to scroll back up.
I would suggest renaming or removing so that every time we read the code we know exactly what are we dealing with.

Reply:
This one i might keep, it keeps convention with other references of moves

Reply:
Can we rename it at least? Just so "moves" are some_moves rather than generic moves. Sorry for being picky.

Reply:
what name did you have in mind? they are the moves in self?

Reply:
Yeah, although at that point it's the same as writing self.move_lines which is why I'm suggesting removing it as it only means someone needs to look up the value of the variable multiple times 😆
It's not that important. I can resolve it if you want to keep it the same way.

Reply:
Thank you ❤️  Sorry for being picky, I'm trying to make these easier to read.